.github/workflows/build.yml

```diff
@@ -89,6 +89,7 @@ jobs:
     name: "Build & Push Containers"
     runs-on: ubuntu-latest
     needs: ["test", "lint", "acceptance-test", "cypress"]
+    if: github.actor != 'dependabot[bot]'
```

Comment:
I don't think we want to skip the whole build in this instance, as the trivy scan is in there, any that's valuable even if it doesn't publish the results.  We don't want a dependabot PR to introduce a vulnerability